pages/event-types/[type].tsx

```diff
@@ -56,9 +57,26 @@ import { DateRangePicker } from "@components/ui/form/DateRangePicker";
 import MinutesField from "@components/ui/form/MinutesField";
 import * as RadioArea from "@components/ui/form/radio-area";
 
+import bloxyApi from "../../web3/dummyResps/bloxyApi";
+
 dayjs.extend(utc);
 dayjs.extend(timezone);
 
+// TODO @danfesi we need to get this boolean from user_settings
+// we should also disable the other API calls for web3 if it's disabled
+const web3App = true;
```

Comment:
@edanfesi can you make sure this is only true if the new `user_settings` table is true?